posthog/models/activity_logging/activity_log.py

```diff
@@ -104,25 +104,35 @@ def log_activity(
     activity: str,
     detail: Detail,
 ) -> None:
-    if activity == "updated" and (detail.changes is None or len(detail.changes) == 0):
-        logger.warn(
-            "'updated' activity was logged with no changes and was ignored",
+    try:
+        if activity == "updated" and (detail.changes is None or len(detail.changes) == 0):
+            logger.warn(
+                "'updated' activity was logged with no changes and was ignored",
+                team_id=team_id,
+                organization_id=organization_id,
+                item_id=item_id,
+                scope=scope,
+            )
+            return
+
+        ActivityLog.objects.create(
+            organization_id=organization_id,
             team_id=team_id,
+            user=user,
+            item_id=str(item_id),
+            scope=scope,
+            activity=activity,
+            detail=detail,
+        )
+    except Exception as e:
+        logger.warn(
```

Comment:
We use Sentry for tracking errors rather than stdout logs, so we should be using `capture_exception` here instead of `logger.warn`.